tests.mustache

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
 <title>Test'em</title>
-<script src="/bower_components/qunit/qunit/qunit.js"></script>
+<script src="/node_modules/qunitjs/qunit/qunit.js"></script>
```

Comment:
Tests are actually fired now. However, some tests seem to fail (CLI report reports 1 failed test, while Chrome browser's result mentions 2 failed tests). They failed before and are not caused by my changes. Please see [this issue](https://github.com/kangax/fabric.js/issues/3404).